egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -43,8 +43,16 @@ def GetArgs():
     # General neural network options
     parser.add_argument("--splice-indexes", type=str, required = True,
                         help="Splice indexes at each layer, e.g. '-3,-2,-1,0,1,2,3'")
+    parser.add_argument("--skip-lda", type=str, action=nnet3_train_lib.StrToBoolAction,
+                        help="skip lda layer", default=False, choices = ["false", "true"])
     parser.add_argument("--include-log-softmax", type=str, action=nnet3_train_lib.StrToBoolAction,
                         help="add the final softmax layer ", default=True, choices = ["false", "true"])
+    parser.add_argument("--add-final-sigmoid", type=str, action=nnet3_train_lib.StrToBoolAction,
```

Comment:
group add-final-sigmoid and objective type options. Checkout how to groups arguments in argparse.